package.json

```diff
@@ -11,14 +11,18 @@
     "dist",
     "src"
   ],
+  "exports": {
+    "./logger": "./dist/logger/index.js",
+    "./eslint": "./dist/logger/index.js"
```

Comment:
Btw. I just saw that this has a wrong path. I'll fix that in a later PR.

Reply:
Good catch